frontend/src/components/Options.vue

```diff
@@ -1,47 +1,143 @@
 <template>
-  <div class="body main-font">
-    <b-navbar-nav>
-      <b-nav-item-dropdown right>
-        <template #button-content>
-          <i class="fa fa-bars"></i>
-        </template>
-        <b-dropdown-group>
+  <div class="body main-font options">
+    <img src="../assets/new-ui/6351903_burger_list_menu_navigation_icon@2x.png" class="burger-icon" @click="openMenu"/>
+    <div class="menu-open" v-if="isMenuOpen">
+      <div @click="closeMenu" class="x-button"><img src="../assets/new-ui/Close.png" /> Close (ESC)</div>
+      <div></div>
+      <div class="ads"></div>
+      <div class="menu-icons">
+        <div class="row-icons">
+          <router-link class="menu-icon" v-if="!stakeOnly" :to="{ name: 'plaza'}" exact>
+            <img src="../assets/new-ui/new-plaza-icon.png">
+            <p>{{ $t("viewLink.character") }}</p>
+          </router-link>
+
+          <router-link class="menu-icon" v-if="!stakeOnly" :to="{ name: 'blacksmith'}" exact>
+            <img src="../assets/new-ui/new-blacksmith-icon.png">
+            <p>{{ $t("viewLink.blacksmith") }}</p>
+          </router-link>
+
+          <router-link class="menu-icon" v-if="!stakeOnly" :to="{ name: 'combat'}" exact>
+            <img src="../assets/new-ui/new-combat-icon.png">
+            <p>{{ $t("viewLink.combat") }}</p>
+          </router-link >
+
+          <router-link class="menu-icon" v-if="pvp" :to="{ name: 'pvp'}" exact>
```

Comment:
Why is arena link here twice? If PvP is not turned on, you can just change the class conditionally. Also, where is the functionality for checking if particular chain supports Quests/PvP/Merch (and informing user about it)?

Reply:
Same for the things below, you can just switch the css class for turned off functionalities